settings/index.js

```diff
@@ -1,28 +1,56 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
 import Settings from '@folio/stripes-components/lib/Settings';
-import { stripesShape } from '@folio/stripes-core/src/Stripes'; // eslint-disable-line import/no-unresolved
 
 import ChangePassword from './ChangePassword';
 
 class MyProfile extends Component {
   static propTypes = {
-    stripes: stripesShape.isRequired,
+    stripes: PropTypes.shape({
+      okapi: PropTypes.shape({
+        ssoEnabled: PropTypes.bool,
+      }).isRequired,
+      intl: PropTypes.shape({
+        formatMessage: PropTypes.func.isRequired,
+      }).isRequired,
+    }).isRequired,
   };
 
   constructor(props) {
     super(props);
 
-    this.pages = [
-      {
-        route: 'password',
-        label: this.props.stripes.intl.formatMessage({ id: 'ui-myprofile.settings.changePassword' }),
-        component: ChangePassword,
-        perm: 'ui-myprofile.view',
-      },
-    ];
+    this.pages = this.createSettingsPages();
+  }
+
+  createSettingsPages() {
```

Comment:
@oleksii-popov What specific comments do you feel are needed? We're not in the habit of commenting every function -- only commenting when there is some specific point to be made.

Reply:
I see: perm, which is not obivious  for me and some other things.

Reply:
I see: perm, which is not obivious  for me and some other things.

Reply:
OK, but `perm` is completely idiomatic in a Stripes module -- short for "permission", of course. It's up to you, since you're the people who will have to maintain this code, but I honestly would not have thought commenting this kind of thing is going to be helpful.